README.md

```diff
@@ -14,6 +14,33 @@ And then execute:
 
 ## Usage
 
+### Automatic error reporting
+
+If you include `govuk_app_config` in your `Gemfile` your application will be automatically configured to send errors to Sentry.
```

Comment:
That makes a lot of sense - https://github.com/alphagov/govuk_app_config/pull/1/commits/b1a3e28daf71435456ba19107c9de1bcf336f50e